securedrop/client.py

```diff
@@ -90,7 +90,12 @@ def register_new_user(self):
 
         pw1 = getpass.getpass(prompt="Enter Password: ")
         pw2 = getpass.getpass(prompt="Re-enter password: ")
+
         if name and email and pw1 and pw2:
+            #enforce password length to min of 12 characters
+            if (len(pw1) < 12) or (len(pw2) < 12):
+                raise RuntimeError("Invalid input")
```

Comment:
Error message should be more descriptive

Reply:
Changed to: "Password is too short! Password must be at least 12 characters"